src/main/templates/facilities.hbs

```diff
@@ -31,10 +25,8 @@
     <tbody>
       {{#data}}
       <tr>
-        <td>
-          <pre>{{code}}</pre>
```

Comment:
I found that the pre tags caused alignment problems in the table. Additionally, I don't think they're necessary as handlebars handles HTML quoting.